### DISCUSSION TITLE
Home Humidity Icon

### DISCUSSION THREAD

**Discussion by cloudbr34k84** [Mariusthvdb/custom-icons#32] [Icon suggestions/requests] [answered]
Im looking to get a Home Humidity Icon. Much like the mdi:home-thermometer icon but with the mdi:water-percent in the middle of home icon.

Much appreciated if this can happen :)

**Comment by therealmcsparrow** [comment, marked as answer]
Added it in my PR
![home-humidity](https://user-images.githubusercontent.com/46250500/184891142-240bfe90-71d7-478f-b3b2-9f55e5c4399c.svg)
R

**Reply by cloudbr34k84** [reply]
thank you, how do i access it?? i dont use github fyi.
